api/src/integrationTest/kotlin/com/cosmotech/api/home/organization/OrganizationConstants.kt

```diff
@@ -30,6 +30,6 @@ object OrganizationConstants {
 
   object Errors {
     val emptyNameOrganizationCreationRequestError =
-        """{"type":"https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400","title":"Bad Request","status":400,"detail":"name: size must be between 1 and 50","instance":"/organizations"}"""
+        """{"type":"https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400","title":"Bad Request","status":400,"detail":"name: size must be between 1 and 2147483647","instance":"/organizations"}"""
```

Comment:
Do we or spring actually checks and enforce that your limit? Or should we change the error message to only say something like "non empty" / "non null"?

Reply:
It's enforced by Jakarta validation (JSON validation) and it's automatic :s (due to openapi minLength usage)
We can try to intercept error message if it really bother you :sweat_smile: